carina_qa/src/test/java/com/solvd/carina/demo/AzureActivityAPITest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2013-2018 QAPROSOFT (http://qaprosoft.com/).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.solvd.carina.demo;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.qaprosoft.apitools.validation.JsonCompareKeywords;
+import com.qaprosoft.carina.core.foundation.AbstractTest;
+import com.qaprosoft.carina.core.foundation.api.http.HttpResponseStatusType;
+import com.qaprosoft.carina.core.foundation.utils.ownership.MethodOwner;
+import com.solvd.carina.demo.api.azure.activities.*;
+import com.solvd.carina.demo.api.bo.AzureActivity;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+
+/**
+ * This sample shows how create REST API tests.
+ *
+ * @author qpsdemo
+ */
+public class AzureActivityAPITest extends AbstractTest {
+
+    @MethodOwner(owner = "k-kudryavtseva")
+    @Test
+    public void testCreateAzureActivity() throws Exception {
+        int id = 0;
+        String title = "sport";
+        String dueDate = "2020-07-28T00:00:00"; //LocalDate.now().toString();
+        boolean completed = true;
+        PostAzureActivityMethod postAzureActivityMethod = new PostAzureActivityMethod(id, title, dueDate, completed);
```

Comment:
let's pass AzureActivityObject into method if you're want to create the while activity. the same for put, patch requests for the rest of the resources (like in DAO)